extensions-core/namespace-lookup/src/main/java/io/druid/server/namespace/URIExtractionNamespaceFunctionFactory.java

```diff
@@ -151,9 +151,23 @@ public String call()
             versionRegex = null;
           }
         } else {
-          final Path filePath = Paths.get(extractionNamespace.getUri());
+          final URI rawURI = extractionNamespace.getUri();
+          final Path filePath = Paths.get(rawURI.getPath());
```

Comment:
This code block only happens if `doSearch` is false; in that case why do we need to be looking at parents at all?